### ISSUE TITLE
Unable to add AWS EC2 Host (Automatically)

### ISSUE THREAD

**Issue by gmeans** [rancher/rancher#1795]
I hooked up a rancher install to our AWS account and tried adding a host via the UI. I can see (via CloudTrail) that it creates the security group, but it never attempts to create the EC2 instance.

Component	Version
Rancher	v0.32.0
Cattle	v0.75.0
User Interface	v0.40.0
Rancher Compose	beta/latest

I was able to manually add a host by launching a RancherOS instance and running the docker command given by the UX.

**Comment by cjellick** [comment]
@gmeans do you see anything in the UI related to the host? Did the status on the host immediately jump to something like "...almost done..."?

Also, can you confirm that the host registration base url is accessible from the host where rancher/server is running. You would have set this value when you first went to add a host and you can get to it from the "Host Registration" menu option in the dropdown in the top right of the UI. Simply grabbing that value, SSHing into the host where rancher/server is running and doing a `curl http://<that value>/v1` should suffice.

**Comment by gmeans** [comment]
@cjellick I'm sorry but I brought down the environment. I'll stand it up again next week. The following is from memory.

- I believe I did see it almost immediately jump to a message that said "...almost done, waiting for agent...". After a bit of time I checked the AWS account and saw no EC2 instance starting up. Then I checked CloudTrail and saw rancher created a security group but never called to create an EC2 instance.

- I did check that this was available, but I'll confirm when I try again.

Is there a container I can tail the logs on and see DEBUG output?
